keepassxc-browser/background/keepass.js

```diff
@@ -806,6 +807,50 @@ keepass.createNewGroup = async function(tab, args = []) {
     }
 };
 
+keepass.getTotp = async function(tab, args = []) {
+    const [ uuid, oldTotp ] = args;
+    if (!keepass.compareVersion('2.6.1', keepass.currentKeePassXC, true)) {
```

Comment:
This must be commented or modified for it to work properly with the current snapshot. The version number is not yet 2.6.1 in the release branch.